test/MUXControlsTestApp/TAEF/Package.VS.appxmanifest

```diff
@@ -25,5 +25,6 @@
   <Capabilities>
     <Capability Name="internetClient" />
     <Capability Name="internetClientServer" />
+    <uap:Capability Name="musicLibrary" />
```

Comment:
When RunTest against a remote machine, after the test, the app is uninstalled, so we can't access the dumped files any more. I need capability to put all dumped files to music folder.

---
In reply to: [254922480](https://github.com/Microsoft/microsoft-ui-xaml/pull/126#discussion_r254922480) [](ancestors = 254922480)

Reply:
Why the music folder? Is there no other way to save out these files?
At the very least this needs to be clearly documented somewhere in the code because it's not at all obvious.

Reply:
We have limited choice: document, music or picture. For me, I don't listen to music so it will not mess up my dev machine. and that's why music folder.
I add some comment to AlwaysLogMasterFile